mkdocs/docs/HPC/FAQ.md

```diff
@@ -226,6 +226,26 @@ information, see .
 
 {% endif %}
 
+
+### Why do I get a "No space left on device" error, while I still have storage space left?
+
+When trying to create files, errors like this can occur:
+
+```shell
+touch: cannot touch 'filename': No space left on device
+```
+
+The error "`No space left on device`" can mean two different things:
```

Comment:
You'll need to add an empty line to make sure list is rendered correctly (also after last bullet)

```suggestion
The error "`No space left on device`" can mean two different things:

```